INSTALL

```diff
@@ -473,7 +473,6 @@ The specific libraries and their options are:
   URL: https://zlib.net
   Options: ZLIB_OPT, ZLIB_LIB.
 
-
```

Comment:
Can we bring back the changes in this file... we will eventually like to make the pure Makefile system work too :)

Reply:
@tzanio as @cwsmith  commented before, this merge doesn't support gnu-make builds with pumi enabled; just cmake. We did check to see that we did not break the mfem gnu-make build (without pumi). That was my fault to modifying INSTALL in the first place.